### PR TITLE
Run sui-surfer in simtest.rs

### PR DESCRIPTION
**Pull request by mystenmark**

### CODE REVIEW

File: crates/sui-benchmark/tests/simtest.rs [hunk 1]
```diff
@@ -531,32 +532,56 @@ mod test {
...
+                )
+                .await
+                .unwrap();
+
+            // TODO: make this stricter (== 0) when we have reliable error retrying on the client.
+            tracing::info!("end of test {:?}", benchmark_stats);
+            assert!(benchmark_stats.num_error_txes < 100);
+        });
+
+        let surfer_task = tokio::spawn(async move {
```
Comment (lxfind) [inline crates/sui-benchmark/tests/simtest.rs:565]: How is this one different from the smoke test in sui-surfer?

Comment (mystenmark) [reply crates/sui-benchmark/tests/simtest.rs:565]: It runs the tests under all the various network conditions already configured in this file (crashes, restarts, reconfig, etc)

File: crates/sui-benchmark/tests/simtest.rs [hunk 2]
```diff
@@ -563,32 +564,56 @@ mod test {
...
+            // now do a sui-surfer test
+            let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+            path.extend([
```
Comment (benr-ml) [inline crates/sui-benchmark/tests/simtest.rs:600]: should we include here all the projects under tests?
I added the tests for Random as another Move project, to be able to isolate its testing

Comment (aschran) [reply crates/sui-benchmark/tests/simtest.rs]: Done

**Review by aschran — APPROVED** [review]

**Pull request merged**